htdocs/user/group/perms.php

```diff
@@ -275,9 +281,12 @@
 	$result = $db->query($sql);
 	if ($result) {
 		$num = $db->num_rows($result);
-		$i = 0;
+		$i = 0;$j = 0;
 		$oldmod = '';
 
+		$cookietohidegroup = (empty($_COOKIE["DOLUSER_PERMS_HIDE_GRP_GRP"]) ? '' : preg_replace('/^,/', '', $_COOKIE["DOLUSER_PERMS_HIDE_GRP_GRP"]));
```

Comment:
For individual permission we use DOLUSER_PERMS_HIDE_GRP
Bu i think we can use the same cookie.  If a user want to hide some module and show other to see permission, the choice will be the same if he is on individual permission or group permission.
Can't we use   DOLUSER_PERMS_HIDE_GRP  as well ?

Reply:
Yes it makes sense